modules/ModularDiffusers/denoise.py

```diff
@@ -63,12 +63,39 @@ class Denoise(NodeBase):
                 {"action": "signal", "target": "guider"},
                 {"action": "signal", "target": "controlnet"},
                 {
-                    "StableDiffusionXLModularPipeline": ["width", "height", "ip_adapter", "controlnet"],
+                    "StableDiffusionXLModularPipeline": [
+                        "width",
+                        "height",
+                        "ip_adapter",
+                        "controlnet",
+                        "latents_preview",
+                    ],
                     "QwenImageModularPipeline": ["width", "height", "controlnet"],
+                    "FluxModularPipeline": [
+                        "width",
+                        "height",
+                        "controlnet",
+                        "ip_adapter",
+                    ],
                     "": [],
                 },
+                {
+                    "action": "value",
+                    "target": "skip_image_size",
+                    "data": {
+                        "QwenImageEditModularPipeline": True,
+                        "QwenImageEditPlusModularPipeline": True,
+                    },
+                },
             ],
         },
+        "skip_image_size": {
+            "label": "Skip Image Size",
+            "type": "boolean",
+            "default": False,
+            "value": False,
+            "hidden": True,
+        },
```

Comment:
"height"/"width" ahows up as 1024/1024 default -can we make it `None`? 
either that or have we a option says "custom output size" 
-> when user click that, we show height/width option, and they can input height/width
->  if they don't click that, we don't show the height/width option and generate with default size

Reply:
the actual UI configuration does not matter to me, but we need to provide a way for them to generate in default size; e.g. qwen sets a default size based on image size and aspetic ratio etc it won't always be 1024 x 1024